pkgs/code_builder/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## 4.11.1-wip
+## 4.11.1
+
+* Convert imports of implementation libraries under `package:fixnum/src/*` into
```

Comment:
Can we also mention that this is expected to be removed in some future release?

Reply:
Hmmmm I'm not sure "removed" is correct, as removing this workaround on this codepath can only be correct if `fixnum` no longer has the different-import-per-platform issue which can only happen with a change to the language e.g. conditional parts. I guess "not any time soon".

Where I expect to end up is that there is new API surface, perhaps in `package:build`, that knows enough to do the right thing without a special case, and we deprecate this whole codepath in favour of the new one.

Not sure if any of that is worth mentioning, what do you think please?

Thanks.